src/screens/Exchange/Exchange/Exchange.js

```diff
@@ -207,6 +207,8 @@ function Exchange() {
               fromInputRef.current = ref;
             }}
             onBlur={() => fromInputRef.current?.blur()}
+            disableAssetChange={false}
```

Comment:
It feels a bit wierd that disable has to be explicitly put to false. Normally you expect boolean value to be either true or false by default rather than default state being inactive.

Reply:
Well the other approach is to add one more prop `forceAssetChange` which basically overrides this and also looks weird, idk :)